mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMessageManager.java

```diff
@@ -44,9 +45,11 @@
 
     public CassandraMessageManager(MailboxSessionMapperFactory mapperFactory, MessageSearchIndex index, 
             MailboxEventDispatcher dispatcher, MailboxPathLocker locker, Mailbox mailbox, QuotaManager quotaManager, 
-            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory) throws MailboxException {
+            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory,
+            FetchBatchSizes fetchBatchSizes) throws MailboxException {
         super(mapperFactory, index, dispatcher, locker, mailbox, new UnionMailboxACLResolver(), 
                 new SimpleGroupMembershipResolver(), quotaManager, quotaRootResolver, messageParser, messageIdFactory);
+        setFetchBatchSizes(fetchBatchSizes);
```

Comment:
If you both agree that fetchBatchSizes is better passed in the constructor then we should remove the setter.
Personnaly I do not see good reason to change batchSizes once the StoreMessageManager is initiated so I agree with @mbaechler.
@aduprat how many implementations of StoreMessageManager are there ?